src/templates/app/_layout/footer.jsx

```diff
@@ -199,73 +199,78 @@ const FooterNormal = () => (
+                        </p>
+                        <p>
+                            {it.L('In the Isle of Man and the UK, Volatility Indices are offered by Binary (IOM) Ltd., First Floor, Millennium House, Victoria Road, Douglas, IM2 4RW, Isle of Man, British Isles; licensed and regulated respectively by (1) the Gambling Supervision Commission in the Isle of Man (current licence issued on 31 August 2017) and by (2) the Gambling Commission in the UK (licence [_1]reference no: 39172[_2]).', '<a href="https://secure.gamblingcommission.gov.uk/PublicRegister/Search/Detail/39172" target="_blank" rel="noopener noreferrer">', '</a>')}
+                        </p>
+                        <p>
+                            {it.L('In the rest of the EU, Volatility Indices are offered by Binary (Europe) Ltd., Mompalao Building, Suite 2, Tower Road, Msida MSD1825, Malta; licensed and regulated by (1) the Malta Gaming Authority in Malta (licence no. MGA/B2C/102/2000 issued on 26 May 2015), for UK clients by (2) the UK Gambling Commission (licence [_1]reference no: 39495[_2]), and for Irish clients by (3) the Revenue Commissioners in Ireland (Remote Bookmaker\'s Licence no. 1010285 issued on 1 July 2017). View complete [_3]Regulatory Information[_2].', '<a href="https://secure.gamblingcommission.gov.uk/PublicRegister/Search/Detail/39495" target="_blank" rel="noopener noreferrer">', '</a>', `<a href="${it.url_for('regulation')}">`)}
+                        </p>
+                    </div>
+                </div>
+                <div className='gr-row'>
+                    <div className='gr-12'>
+                        <div className='about-binary'>
+                            <p>
+                                {it.L('Binary.com is an award-winning online trading provider that helps its clients to trade on financial markets through binary options and CFDs. Trading binary options and CFDs on Volatility Indices is classified as a gambling activity. Remember that gambling can be addictive – please play responsibly. Learn more about [_1]Responsible Trading[_2]. Some products are not available in all countries. This website’s services are not made available in certain countries such as the USA, Canada, Costa Rica, Hong Kong, or to persons under age 18.', `<a href="${it.url_for('responsible-trading')}">`, '</a>')}
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div className='gr-row'>
+                    <div className='gr-12'>
+                        <div className='risk-warning'>
+                            <p>
+                                {it.L('Trading binary options may not be suitable for everyone, so please ensure that you fully understand the risks involved. Your losses can exceed your initial deposit and you do not own or have any interest in the underlying asset.')}
+                            </p>
+                            <p>
```

Comment:
This should be only displayed to EU countries

Reply:
isn't this the default footer? i think this sentences should be visible to all clients 🤔

Reply:
@ashkanx @kellybinary yes, it is visible for all in staging. Only CFD warning is EU exclusive.